src/xbot/common/controls/sensors/BaseXboxControllerAdapter.java

```diff
@@ -0,0 +1,211 @@
+package xbot.common.controls.sensors;
```

Comment:
having the name Base and Adapter is kind of rough, maybe just `XboxControllerAdapter`